jupyter_server_proxy/handlers.py

```diff
@@ -242,27 +245,27 @@ def proxy_request_options(self):
     # Support all the methods that torando does by default except for GET which
     # is passed to WebSocketHandlerMixin and then to WebSocketHandler.
 
-    async def http_get(self, port, proxy_path=''):
+    async def http_get(self, host, port, proxy_path=''):
```

Comment:
I *think* we should remove all these definitions - this handler should only be used by inherited handlers, and these need to be manually defined there anyway. And where they are defined manually, you must not let the user define which host it is...

Reply:
Interesting idea. I'm a little reticent to do something to break this class in such an implicit way -- that will make developing and debugging the subclassed handler trickier when there is no indication of what methods to override.

What if instead we make all these functions raise an error with a useful message?

Reply:
Okay, I've taken a crack at what I suggested. Let me know what you think.